examples/mongodb/README.md

```diff
@@ -1,7 +1,71 @@
 This example is desgined to be run on a mongo replicaset.
 
- * Deploy 5 mongodb units
- * Run main.go specifying the urls to use
- * It will tell you which url is the primary node
- * Use part.sh to cause a partition around the specified unit while main.go is running. Set Primary to the primary unit (ip address output from main.go)
- * See if the expected len is different from what is found in the db.
+### Deploy 5 mongodb units
+
+juju deploy mongodb -n 5
```

Comment:
The mongodb charm docs mention potential issues with doing this. Might be worth making a note of them here.